### PR TITLE
Port obs_lsstSim to python3 Tickets/dm 7305

### PR DESCRIPTION
**Pull request by parejkoj**

### CODE REVIEW

File: config/processCcd.py [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 config.isr.retarget(LsstSimIsrTask)
 
 try:
     import lsst.meas.extensions.multiShapelet
-    config.calibrate.measurement.algorithms.names = set(config.measurement.algorithms.names) | lsst.meas.extensions.multiShapelet.algorithms
+    config.calibrate.measurement.algorithms.names = set(
+        config.measurement.algorithms.names) | lsst.meas.extensions.multiShapelet.algorithms
     config.calibrate.measurement.slots.modelFlux = "multishapelet.combo.flux"
 except ImportError:
     # TODO: find a better way to log this
```
Comment (mrawls) [inline config/processCcd.py]: 😆

File: python/lsst/obs/lsstSim/eimageIsr.py [hunk 2]
```diff
@@ -92,16 +92,16 @@ def runDataRef(self, sensorRef):
...
         # Interpolate
         isr.interpolateFromMask(
-            maskedImage = mi,
-            fwhm = self.config.interp_size,
-            growFootprints = 0,
-            maskName = 'SAT',
+            maskedImage=mi,
+            fwhm=self.config.interp_size,
+            growFootprints=0,
+            maskName='SAT',
```
Comment (mrawls) [inline python/lsst/obs/lsstSim/eimageIsr.py]: Spaces around equals signs should be preserved here too.

File: python/lsst/obs/lsstSim/lsstSimIsrTask.py [hunk 3]
```diff
@@ -159,7 +159,7 @@ def runDataRef(self, sensorRef):
             getDisplay(frame).mtv(postIsrExposure)
 
         return pipeBase.Struct(
-            exposure = postIsrExposure,
+            exposure=postIsrExposure,
```
Comment (mrawls) [inline python/lsst/obs/lsstSim/lsstSimIsrTask.py]: Spacing

File: python/lsst/obs/lsstSim/processCalibLsstSim.py [hunk 4]
```diff
@@ -120,17 +122,17 @@ def run(self, sensorRefList, calibType):
...
-                butler.put(exp, calibType, dataId = amp.dataId)
+                print("writing file %s"%dataId)
+                butler.put(exp, calibType, dataId=amp.dataId)
             masterExpList.append(exp)
             dataIdList.append(amp.dataId)
         if self.config.doWrite and calibType is 'flat':
             self.normChipAmps(masterExpList)
             for exp, dataId in zip(masterExpList, dataIdList):
-                print "writing flat file %s"%dataId
+                print("writing flat file %s"%dataId)
```
Comment (mrawls) [inline python/lsst/obs/lsstSim/processCalibLsstSim.py]: The % symbol should have spaces around it when used for formatting print statements. And more equals signs just below this too!

File: python/lsst/obs/lsstSim/selectFluxMag0.py [hunk 5]
```diff
@@ -152,5 +153,5 @@ def runArgDictFromDataId(self, dataId):
         @return keyword arguments for visit (other than coordList), as a dict
         """
         return dict(
-            visit = dataId["visit"]
+            visit=dataId["visit"]
```
Comment (mrawls) [inline python/lsst/obs/lsstSim/selectFluxMag0.py]: MOAR equals sign spacings

File: python/lsst/obs/lsstSim/selectLsstImages.py [hunk 6]
```diff
@@ -38,9 +40,9 @@ class SelectLsstImagesConfig(DatabaseSelectImagesConfig):
     """Config for SelectLsstImagesTask
     """
     maxFwhm = pexConfig.Field(
-        doc = "maximum FWHM (arcsec)",
-        dtype = float,
-        default = 2.0,
+        doc="maximum FWHM (arcsec)",
+        dtype=float,
+        default=2.0,
```
Comment (mrawls) [inline python/lsst/obs/lsstSim/selectLsstImages.py]: Lots of equals signs throughout need spaceifying

**Comment by mrawls** [comment]
All right, I retract my comment about whitespace around equals signs, because apparently that is one of those non-pep8-things we *have* been doing but would like to eventually *not* do. That is very unclear from the style guide 😒

File: python/lsst/obs/lsstSim/processCalibLsstSim.py (hunk 4 above)
Comment (parejkoj) [inline python/lsst/obs/lsstSim/processCalibLsstSim.py]: Percent fixed.

File: config/processCcd.py (hunk 1 above)
Comment (parejkoj) [inline config/processCcd.py]: Indeed, especially with your stderr comment above...

**Comment by parejkoj** [comment]
Just so its clear, the whitespace thing is probably going away in the future, per the discussion on [DM-5456](https://jira.lsstcorp.org/browse/DM-5456?focusedCommentId=53306&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-53306) and the associated [RFC-162](https://jira.lsstcorp.org/browse/RFC-162). Since autopep8 formatted it as above, we might as well leave it.

**Pull request merged**